.github/workflows/ci.yml

```diff
@@ -32,18 +32,25 @@ jobs:
           source github-actions-setup.sh nightly-x64
           make test
 
-  macos:
-    runs-on: macos-11
-    steps:
-      - uses: actions/checkout@v2
-      - name: neovim
-        run: brew install neovim --HEAD
+  # macos:
```

Comment:
you know this here is running nvim tests for both ubuntu and mac at the same time

https://github.com/nvim-telescope/telescope-fzf-native.nvim/blob/2fd60ebe4c372199e0d953310640b1aeaf06166f/.github/workflows/ci.yml#L52-L88

Reply:
thats what i meant with matrix.
have it in the same definition

Reply:
Thanks conni ❤️